package.json

```diff
@@ -22,6 +22,14 @@
   },
   "homepage": "https://github.com/Ian-Stewart-Binks/foosball#readme",
   "devDependencies": {
-    "express": "^4.13.3"
+    "express": "^4.13.3",
```

Comment:
"express" should be in dependencies not devDependencies since its required for production use. Probably same for "react-bootstrap" below. Also I don't think you need to repeat "react" and "react-dom" in devDependencies